config/locales/en.yml

```diff
@@ -89,3 +89,24 @@ en:
             attributes:
               user:
                 too_many_reviewers: too many reviewers
+          application_draft:
+            attributes:
+              projects:
+                too_many_selected: must not be selected twice
+                none_accepted: must have been accepted
+              base:
+                too_many_applications: Only one application may be lodged
+                unconfirmed_email_address: Please make sure every student confirmed the email address.
+          conference:
+            attributes:
+              ends_on:
+                chronology_violation: must be a later date than start date
+          team:
+            attributes:
+              roles:
+                too_many_students: there cannot be more than 2 students on a team.
```

Comment:
For the team, two base errors are missing here. (line 175 and 183)
https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/a020e98eb1339e29170fdb7e7c64404fe0719b2c/app/models/team.rb#L165-L184

It is a bit more challenging than the other errors. Are you up to that?